collections-lists-and-sets/src/main/java/skyprocollectionslistsandsets/controller/DepartmentAndSalaryEmployeeController.java

```diff
@@ -0,0 +1,42 @@
+package skyprocollectionslistsandsets.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import skyprocollectionslistsandsets.employees.Employee;
+import skyprocollectionslistsandsets.service.DepartmentAndSalaryEmployeeService;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping(path = "employee/departments")
```

Comment:
```suggestion
@RequestMapping(path = "/departments")
```
по заданию просто /departments путь